### PR TITLE
Make date range fields stack horizontally.

### PR DESCRIPTION
**Pull request by webinista**
Closes department-of-veterans-affairs/messaging-fe#187

- Removed some unnecessary CSS.
- Adjusted flex value for date range legend.
- Adjusted padding for span, and flex layout of date range.

Now looks like this. 

screen shot 2016-12-12 at 7 35 42 pm

### CODE REVIEW

**Comment by webinista** [comment]
@angel Added you as a reviewer for a design check.

**Comment by webinista** [comment]
@angel Can you specify an exact amount of space?

File: src/sass/messaging/partials/_messaging-search.scss [hunk 1]
```diff
@@ -130,54 +133,38 @@
...
-    @media (min-width: $medium-screen) {
-      display: flex;
-      align-items: center;
-    }
+    padding-left: .5rem;
+    padding-right: .5rem;
   }
 }
 
 .react-datepicker__input-container {
```
Comment (U-DON) [inline src/sass/messaging/partials/_messaging-search.scss]: Can probably remove this since it's empty now.

**Comment by angel** [comment]
@webinista 

1rem between the form field and the exact match
1em below the checkbox
2.5rem between the date range form field and button

To look like this:
image

**Comment by U-DON** [comment]
LGTM.

**Pull request merged**